### ISSUE TITLE
Change Auto-WCAG meeting times

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#397]
Proposal on the table is to go to a bi-monthly call:

- 2nd Thursday of the month, 16:00 - 17:00 CET
- 4th Thursday of the month, 10:00 - 11:00 CET

I'm hoping this change will enable more participation from Asia and Australia. It also increases the total time for calls to 2 hours per month instead of 1:30.

**Open question**: What do we do with the agenda? Do we have a shared agenda for both meetings, do we have some shared items, or do we roll over the agenda?

**Comment by nitedog** [comment]
I propose taking minutes for every meeting, and people who miss a meeting (for whatever reason) are expected to read the minutes to remain current with the progress of the group. This should hopefully minimize repetition. Important decisions should be sent to the list with a few days grace period, so that people who miss a meeting (again, for whatever reason) have a chance to voice their opinion.

**Comment by EmmaJP** [comment]
I think a rolling agenda across both meetings could work OK. If something has enough input in one meeting it can be actioned/resolved, and if not it can roll to the other meeting for additional eyeballs and comments. My low contributor 2p.

**Comment by annethyme** [comment]
@WilcoFiers, is this one done and can be closed?

**Comment by jkodu** [comment]
This is completed.
